.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+**This is a:** Bug fix / New feature / Rewrite / Clean-up.
+
+**I need testing:** Yes (explain) / No.
+
+**Perceived status:** Ready / Unsure / Work in progress / I need help.
```

Comment:
Could "I need help" and "I need testing" be merged into one section? ("I need assistance"?) 

WIP/Ready could be decided by setting the PR to a draft or not.